src/proteomes/config/__tests__/ProteomesColumnConfiguration.spec.tsx

```diff
@@ -23,7 +21,7 @@ describe('ProteomesColumnConfiguration component', () => {
     `should render column "%s"`,
     (key, column) => {
       const { asFragment } = customRender(
-        <MemoryRouter>{column.render(transformedData)}</MemoryRouter>
+        <>{column.render(transformedData)}</>
```

Comment:
It's because `customRender` accepts only a `ReactElement`. I could have made it accept a `ReactNode` but I thought it would be better if it matched the signature of the underlying `render` which also only accepts `ReactElement`s